map.js

```diff
@@ -0,0 +1,96 @@
+// GEOLOCATION
+async function getCoords(){
+	const position = await new Promise((resolve, reject) => {
+		navigator.geolocation.getCurrentPosition(resolve, reject)});
+	return [position.coords.latitude, position.coords.longitude]}
+
+    // function(getIcon)  This was supposed to be a my custome marker icon! I got stuck on fetching the proper data from the server :( I just need to review it more - German Almonte
```

Comment:
I appreciate the comment + explanation! If you ever want help figuring this out let me know :)